backend/src/routes/criteria.ts

```diff
@@ -0,0 +1,42 @@
+import { Router } from 'express';
+import Criteria from 'models/criteria';
+
+const router = Router();
+
+router.get('/', (req, res) => {
+  Criteria.find({}, (err, product) => {
+    if (err) res.send('Error getting all Criteria');
+    else res.send(product);
```

Comment:
I think we should be explicit with all of our status codes... what is the default, 200?